tests/modules/nf-scil/registration/antsapplytransforms/main.nf

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { LOAD_TEST_DATA } from '../../../../../subworkflows/nf-scil/load_test_data/main.nf'
+include {
+    REGISTRATION_ANTSAPPLYTRANSFORMS as REGISTRATION_ANTSAPPLYTRANSFORMS_TEST;} from '../../../../../modules/nf-scil/registration/antsapplytransforms/main.nf'
+
+workflow test_registration_applytransforms {
+
+    input_fetch = Channel.from( [ "bst.zip" ] )
+
```

Comment:
I had Alexa add a folder for registration tests. Change the bst for this folder, see:

https://github.com/scilus/nf-scil/pull/86/files#diff-bfd81ff7b07a233fa211a66a9ffd676a3fc25bf7ce64d84b742231d2b2f0d7e5